Homework_python_3/Task_03.py

```diff
@@ -0,0 +1,28 @@
+# Задайте список из вещественных чисел. Напишите программу,
+# которая найдёт разницу между максимальным и минимальным значением дробной части элементов.
+# Пример:
+# [1.1, 1.2, 3.1, 5, 10.01] => 0.2
+import math
+lst = [1.02, 2.56, 3.589, 4.3255, 5.34743, 6.98795, 5, 7]
+min = 0.9
+max = 0
+for i in range(len(lst)):
+    a = lst[i] - math.floor(lst[i])
```

Comment:
```suggestion
    a = lst[i] - int((lst[i]))
```
Так ведь проще, не правда ли?